synapse/config/federation.py

```diff
@@ -65,5 +65,19 @@ def read_config(self, config: JsonDict, **kwargs: Any) -> None:
         self.max_long_retries = federation_config.get("max_long_retries", 10)
         self.max_short_retries = federation_config.get("max_short_retries", 3)
 
+        # Allow for the configuration of the backoff algorithm used
+        # when trying to reach an unavailable destination.
+        # Unlike previous configuration those values applies across
+        # multiple requests and the state of the backoff is stored on DB.
+        self.destination_min_retry_interval_ms = Config.parse_duration(
+            federation_config.get("destination_min_retry_interval", "10m")
+        )
+        self.destination_retry_multiplier = federation_config.get(
+            "destination_retry_multiplier", 2
+        )
+        self.destination_max_retry_interval_ms = Config.parse_duration(
+            federation_config.get("destination_max_retry_interval", "7d")
+        )
```

Comment:
I *think* that this needs a true "cap" on what to allow otherwise it won't allow fitting into the database? See #6146.

Reply:
Good catch, done.